src/lib/Behat/Page/ContentUpdateItemPage.php

```diff
@@ -48,6 +48,12 @@ public function __construct(
 
     public function verifyIsLoaded(): void
     {
+        $isIFrameExisting = $this->getHTMLPage()->findAll(new VisibleCSSLocator('iframe', '.c-content-edit__iframe'))->any();
+        if ($isIFrameExisting) {
+            $this->getHTMLPage()->executeJavaScript("document.querySelector('.c-content-edit__iframe').setAttribute('name','value')");
+            $this->getSession()->switchToIFrame('value');
+        }
```

Comment:
```suggestion
        $iframeLocator = new VisibleCSSLocator('iframe', '.c-content-edit__iframe');
        $isIFrameExisting = $this->getHTMLPage()->setTimeout(0)->findAll($iframeLocator)->any();
        if ($isIFrameExisting) {
            $script = sprintf("document.querySelector('%s').setAttribute('name','editIframe')", $iframeLocator->getSelector());
            $this->getHTMLPage()->executeJavaScript($script);
            $this->getSession()->switchToIFrame('editIframe');
        }
```

1) I'm setting the timeout to 0 so that the tests which don't have the iframe present don't have to wait 1 second to continue execution (currently they would be blocked, waiting 1 second for the iframe to appear)
2) Something like `editIframe` is a better name for the iframe than `value`
3) I've extracted the locator to a separate variable to avoid repeating it in the JS

Also: we need a block like this:
```
        if ($isIFrameExisting) {
            $this->getSession()->switchToIFrame();
        }
```
after all the checks have been done - that's the usual way of working with iframes, we need to switch back so that the code that is executing after is using the default frame level.